@here/harp-examples/index.html

```diff
@@ -1,65 +1,367 @@
+                bottom: 0;
+                width: 100%
+            }
+
+            #exampleList div {
+                width: 100%;
+            }
+
+            #exampleList div h2 {
+                margin: 20px 20px 5px 15px;
+                padding-bottom: 5px;
+                font-size: 16px;
+                color: #b1bac9;
+                text-transform: uppercase
+            }
+
+            #exampleList div:first-of-type h2 {
+                margin-top: 0
+            }
+
+            #exampleList .link {
+                text-decoration: none;
+                cursor: pointer;
+                display: block;
+                color: #d2d7df;
+                padding: 3px 0 3px 15px;
+            }
+
+            #exampleList .selected {
+                font-weight: bold;
+                background: #2c333f
+            }
+
+            #exampleList .link:hover {
+                background: #2c333f
+            }
+
+            #exampleList .filtered {
+                display: none !important;
+            }
+
+            /*
+            * View Source button
+            */
+            #viewSource {
+                width: 40px;
+                height: 40px;
+                font-size: 22px;
+                font-weight: bold;
+                outline: none;
+                border: none;
+                color: rgba(255, 255, 255, 0.8);
+                cursor: pointer;
+                border-radius: 4px;
+                box-shadow: 0 1px 4px 0 rgba(15, 22, 33, 0.4);
+                transition: all 0.1s;
+                padding: 0 0 1px 1px;
+                user-select: none;
+                position: fixed;
+                bottom: 25px;
+                right: 15px;
+                background: #272d37;
+                text-decoration: none;
+            }
+
+            #viewSource p {
+                font-size: 15px;
+                font-weight: bold;
+                margin: 0;
+            }
+
+            /*
+            * Filter input block
+            */
+
+            .filterBlock {
+                position: relative;
+                margin: 0px 0 15px 0;
+            }
+
+            .filterBlock p {
+                margin: 0;
+            }
+
+            #filterInput {
+                background: #d2d7df;
+                width: 100%;
+                box-sizing: border-box;
+                padding: 20px 55px 20px 20px;
+                color: #1f242d;
+                font-family: inherit;
+                font-size: 15px;
+                outline: none;
+                border: 1px solid #4e5a6e;
+                border-left: none;
+                border-right: none;
+            }
+
+            #magnifier-placeholder {
+                position: absolute;
+                top: 22px;
+                left: 16px;
+                pointer-events: none;
+                font-size: 25px;
+                transform: rotateZ(-45deg);
+                color: #4e5a6e;
+            }
+
```

Comment:
You can also hide icon when search field gains focus
```suggestion

#filterInput:focus + #magnifier-placeholder {
    display: none;
}
```